src/main/java/com/blogify/blogapi/constant/FileConstant.java

```diff
@@ -0,0 +1,7 @@
+package com.blogify.blogapi.constant;
+
+import java.time.Duration;
+
+public class FileConstant {
+  public static final Duration URL_DURATION = Duration.ofMinutes(2);
```

Comment:
You can put it in FileUserService or do we need a lot of file constant?

Reply:
It's used by UserFileService and PostFileService